robocop/checkers/misc.py

```diff
@@ -270,34 +269,49 @@ class ReturnChecker(VisitorChecker):
 
     reports = (
         "keyword-after-return",
-        "keyword-after-return-from",
         "empty-return",
     )
 
     def visit_Keyword(self, node):  # noqa
         return_setting_node = None
         keyword_after_return = False
         return_from = False
+        error = ""
         for child in node.body:
             if isinstance(child, Return):
                 return_setting_node = child
+                error = (
+                    "[Return] is not defined at the end of keyword. "
+                    "Note that [Return] does not return from keyword but only set returned variables"
```

Comment:
I guess that would make more sense:
```suggestion
                    "Note that [Return] does not quit from keyword but only set variables to be returned"
```